reqs/requirements-dev-lite.txt

```diff
@@ -5,6 +5,7 @@ freezegun>=0.3.15
 ipykernel<=6.17.1 # Newest version (6.19.0 released on 12/7/2022) causes "WARNING  traitlets:client.py:1181 No handler found for comm target 'comm'" by "https://github.com/jupyter/nbclient/blob/main/nbclient/client.py" (version 0.7.2) to be emitted.
 moto>=2.0.0,<3.0.0
 nbconvert>=5
+pact-python>=2.0.1
```

Comment:
This is a testing dependency right? Should it go in requirements-dev-test or whatever equivalent file we have?

Reply:
`requirements-dev-test.txt` uses this file. Other testing deps are in here like `pytest` and `snapshottest`.